flyway/src/test/groovy/io/micronaut/configuration/dbmigration/flyway/FlywayConfigurationPropertiesSpec.groovy

```diff
@@ -134,4 +134,28 @@ class FlywayConfigurationPropertiesSpec extends Specification {
         cleanup:
         applicationContext.close()
     }
+
+    void 'test define flyway database connection via JDBC only and not use Micronaut datasource'() {
+        given:
+        EmbeddedServer server = ApplicationContext.run(EmbeddedServer,
+            ['spec.name'                         : FlywayConfigurationPropertiesSpec.simpleName,
+             'flyway.datasources.books.locations': 'classpath:databasemigrations',
+             'flyway.datasources.books.url'      : 'jdbc:h2:mem:flywayBooksDB3;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;USER=sa',
+             'flyway.datasources.books.user'     : '',
+             'flyway.datasources.books.password' : '',
```

Comment:
Please remove `flyway.datasources.books.user` and `flyway.datasources.books.password`